pangeo_forge_orchestrator/models.py

```diff
@@ -81,6 +125,67 @@ class RecipeRunRead(RecipeRunBase):
     id: int
 
 
-MODELS = {
-    "recipe_run": MultipleModels(path="/recipe_runs/", base=RecipeRunBase, response=RecipeRunRead)
-}
+# Extended response models --------------------------------------------------------------
+# https://sqlmodel.tiangolo.com/tutorial/fastapi/relationships/#models-with-relationships
+
+
+class BakeryReadWithRecipeRuns(BakeryRead):
+    recipe_runs: List[RecipeRunRead]
+
+
+class FeedstockReadWithRecipeRuns(FeedstockRead):
+    recipe_runs: List[RecipeRunRead]
+
+
+class RecipeRunReadWithBakeryAndFeedstock(RecipeRunRead):
+    bakery: BakeryRead
+    feedstock: FeedstockRead
+
+
+# Mutliple models -----------------------------------------------------------------------
+
+
+bakery_models = MultipleModels(
+    path="/bakeries/",
+    base=BakeryBase,
+    response=BakeryRead,
+    extended_response=BakeryReadWithRecipeRuns,
+    relations=[
+        RelationBuilder(
+            field="recipe_runs",
+            back_populates="RecipeRun.bakery",
```

Comment:
@rabernat, note here that I'm passing `back_populates` as a dot-delimited string, which is then [`split()` in `model_builders.MultipleModels.make_table_cls`](https://github.com/pangeo-forge/pangeo-forge-orchestrator/blob/fa0392fd3d29b15b124b174fba779e0fe14c66d3/pangeo_forge_orchestrator/model_builders.py#L128).

`sqlmodel.Relationship`, [for which this is a kwarg value](https://sqlmodel.tiangolo.com/tutorial/relationship-attributes/back-populates/), just wants "the name of the attribute in the other model class" (i.e. "bakery"), not a string inclusive of the name of the other model class (i.e. "RecipeRun.bakery").

But for some reason, I just find this _very_ difficult to read without including the model name in the string. I don't love doing this, because it breaks @tmcw's guidance to ["call things the same thing"](https://macwright.com/2021/02/17/the-naming-of-things.html).

Curious how this reads to you. If we keep it this way, I can clarify what we're looking for in [`RelationBuilder` docstring](https://github.com/pangeo-forge/pangeo-forge-orchestrator/blob/fa0392fd3d29b15b124b174fba779e0fe14c66d3/pangeo_forge_orchestrator/model_builders.py#L17). Open to changing it though.

Reply:
If the value is being ignored by the code that consumes it, then it should be removed.

Is the `RecipeRun` part of `RecipeRun.bakery` always the same value `annotations=List["RecipeRun"]`. If so, it seems like we could avoid this redundancy.

Reply:
> If the value is being ignored by the code that consumes it, then it should be removed.

Yes... it is being ignored. 🤭 

> Is the RecipeRun part of RecipeRun.bakery always the same value annotations=List["RecipeRun"]. If so, it seems like we could avoid this redundancy.

Correct. The `back_populates` field will always be field of the model referenced in `annotations`. This is actually obvious now that I write it out that way, but somehow felt more elusive before. I will fix this to eliminate the redundancy and update the docstring accordingly.

Reply:
https://github.com/pangeo-forge/pangeo-forge-orchestrator/pull/31/commits/56c0323001c7d3cfb4663486aa0cccd1c47b17fa